src/main/java/frc/robot/subsystems/IndexerSubsystem.java

```diff
@@ -0,0 +1,46 @@
+package frc.robot.subsystems;
+
+import static frc.robot.Constants.IndexerConstants.DEVICE_ID_BELT;
+
+import com.ctre.phoenix6.controls.VelocityTorqueCurrentFOC;
+import com.ctre.phoenix6.hardware.TalonFX;
+import edu.wpi.first.wpilibj2.command.Subsystem;
+import frc.robot.Constants;
+
+public class IndexerSubsystem implements Subsystem {
+
+  private final TalonFX beltMotor = new TalonFX(DEVICE_ID_BELT);
+  private final VelocityTorqueCurrentFOC beltControl = new VelocityTorqueCurrentFOC(0.0);
+
+  public IndexerSubsystem() {
+
+  }
+
+  /**
+   * runs belt to move coral onto end effector
```

Comment:
Let's start JavaDoc comments with a capital letter, they should be a sentence (no period needed, unless there are multiple sentences). This applies to all of the JavaDoc comments